.github/actions/javascript/getPreviousVersion/getPreviousVersion.js

```diff
@@ -0,0 +1,14 @@
+const {readFileSync} = require('fs');
+const core = require('@actions/core');
+const _ = require('underscore');
+const versionUpdater = require('../../../libs/versionUpdater');
+
+let semverLevel = core.getInput('SEMVER_LEVEL', {require: true});
+if (!semverLevel || !_.contains(versionUpdater.SEMANTIC_VERSION_LEVELS, semverLevel)) {
+    semverLevel = versionUpdater.SEMANTIC_VERSION_LEVELS.PATCH;
```

Comment:
Why do we default to `PATCH` specifically here?

Reply:
I guess there's no great reason since it's a required input. Updated this so that it just throws and error if the semver level isn't valid.